modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/ClaimManagementRestClient.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.integration.test.restclients;
+
+import io.restassured.http.ContentType;
+import org.apache.commons.codec.binary.Base64;
+import org.apache.http.Header;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.util.EntityUtils;
+import org.json.simple.JSONObject;
+import org.testng.Assert;
+import org.wso2.carbon.automation.engine.context.beans.Tenant;
+import org.wso2.identity.integration.test.rest.api.server.claim.management.v1.model.ExternalClaimReq;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ClaimManagementRestClient extends RestBaseClient {
+
+    private static final String TENANT_PATH = "t/%s";
+    private static final String API_SERVER_BASE_PATH = "/api/server/v1";
+    private static final String CLAIM_DIALECTS_ENDPOINT_URI = "/claim-dialects";
+
+    public static final String CLAIMS_ENDPOINT_URI = "/claims";
+    private final CloseableHttpClient client;
+    private final String username;
+    private final String password;
+    private final String serverBasePath;
+
+    public ClaimManagementRestClient(String backendURL, Tenant tenantInfo) {
+        client = HttpClients.createDefault();
```

Comment:
Shall we add the missing new line here?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096